developers/agents/_includes/transformation_agent_tutorial_enrich_dataset.py

```diff
@@ -46,10 +46,9 @@
 
 papers_collection = client.collections.get("ArxivPapers")
 
-with papers_collection.batch.dynamic() as batch:
-    for i, item in enumerate(dataset):
-        if i < 200:
-            batch.add_object(properties=item["properties"])
+with papers_collection.batch.fixed_size(batch_size=200) as batch:
+    for item in dataset:
```

Comment:
Heya, can we add the limit back in? I think I added it to speed up these examples.

Reply:
Good point, adding it back